numba/tests/cffi_usecases.py

```diff
@@ -51,9 +53,12 @@
     ffi_ool = FFI()
     ffi.set_source('cffi_usecases_ool', source)
     ffi.cdef(defs, override=True)
-    ffi.compile()
-
+    tmpdir = static_temp_directory('test_cffi')
+    ffi.compile(tmpdir=tmpdir)
+    sys.path.append(tmpdir)
```

Comment:
It would be better to use a `try .. finally` block in case the module import fails.